src/components/_basic.less

```diff
@@ -72,3 +72,12 @@ a.sbtn {
 .purple-btn {
   .colorRule(@purple);
 }
+
+.white-btn {
+  .colorRule(@white);
+
+  &:focus {
+    box-shadow: 0px 0px 0 0.2rem fade(@darkText, 30%) !important;
```

Comment:
don't use `@darkText` add a new variable with the color `#bababa` and use it. Also, I tested it and I don't think using important here is necessary.

Reply:
I updated to suggested change! Please review! thanks.